src/app/GameController.ts

```diff
@@ -36,9 +36,9 @@ export class GameController {
     this.gameProcessor.pauseTurn();
   }
 
-  turnAlert(): void {
-    this.view.turnAlert();
-  }
+  // turnAlert(): void {
+  // this.view.turnAlert();
+  // }
```

Comment:
I guess this method might be useful for passing the alerts to the alert component. It's @Enessetere's task.